frontend/components/TableContainer/DataTable/DataTable.tsx

```diff
@@ -583,30 +583,16 @@ const DataTable = ({
                   {row.cells.map((cell: any, index: number) => {
                     // Only allow row click behavior on first cell
                     // if the first cell is not a checkbox
-                    const isFirstCell = index === 0;
                     const cellProps = cell.getCellProps();
-
```

Comment:
Hey @RachelElysia, I'm testing these changes as Dante made them and tables seem to be working okay - still tab-able, enter checks and unchecks boxes, and I can click through for rows that have row-click functionality. The bug it targets for fixing is also fixed.

Since you wrote this pretty recently, can you clarify the intended behavior of these lines, and confirm that it's okay to remove them like this?

Reply:
hey all. I ended up not make a change to enableMutliRowSelect. I will take a look at the PR now